bin/xdebug-enable

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+sudo rm -f /usr/local/etc/php/conf.d/*blackfire.ini
```

Comment:
would be great to enable and disable blackfire too

Reply:
@ChrisB9 you can add commands for that. It is a little bit trickier, as there is more to it than just set `zend_extension=xdebug.so`. This PR is only for xdebug 😄

Reply:
true - just as a future idea